src/components/include/utils/logger.h

```diff
@@ -141,6 +143,8 @@ log4cxx_time_t time_now();
 
 #define DEINIT_LOGGER()
 
+#define DELETE_LOGGER()
```

Comment:
@SNiukalov  please add some description to this macro

Reply:
@LuxoftAKutsan 
Fixed in:https://github.com/smartdevicelink/sdl_core/pull/3022/commits/5b3242f2ab66d30ea0c5b71bad649e3dabccb516